packages/sx.js/src/highlightConstants.ts

```diff
@@ -3,6 +3,18 @@ export const HIGHLIGHT_DOMAIN = {
   version: '0.1.0'
 };
 
+export const HIGHLIGHT_STARKNET_DOMAIN = {
+  ...HIGHLIGHT_DOMAIN,
+  revision: '1'
```

Comment:
This should be integer afaik:

From the [spec](https://github.com/starknet-io/SNIPs/blob/main/SNIPS/snip-12.md):
> Revision `1`: Will be the initial version of the specification. Note that for this revision the value in this field should be the integer 1 and not the shortstring "1" despite being defined as shortstring. This exception is made to support an inconsistency in the Braavos wallet implementation.

Example: https://github.com/starknet-io/SNIPs/blob/main/SNIPS/snip-12.md#json-example

Reply:
Starknet v6 will only accept a string, and return a "wrong format" error when using a number.

Upgraded the dependency to 7.6.4 to match the master branch to make it work